### PR TITLE
Remove @react-native-community/art dependency

### PR DESCRIPTION
**Pull request by swansontec**
Since we are using react-native-svg, we don't need a second native graphics library.

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [ ] Tested on small Android
- [ ] n/a

### CODE REVIEW

File: patches/edge-login-ui-rn+0.9.32.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@@ -14,6 +14,7 @@ export function QrCode(props) {
+     const path = svg.replace(/.*d="([^"]*)".*/, '$1');
+     // Create a drawing transform to scale QR cells to device pixels:
+     const sizeInCells = code.getModuleCount() + 2 * padding;
+-    return (React.createElement(Surface, { height: size, width: size, style: { backgroundColor } },
+-        React.createElement(Shape, { d: path, fill: foregroundColor, scale: size / sizeInCells })));
++    const viewBox = `0 0 ${sizeInCells} ${sizeInCells}`
++    return (React.createElement(Svg, { height: size, width: size, style: { backgroundColor }, viewBox: viewBox },
++        React.createElement(Path, { d: path, fill: foregroundColor })));
+ }
```
Comment (thehobbit85) [inline patches/edge-login-ui-rn+0.9.32.patch:20]: Is there a reason why we don't just fix it directly in edge-login-ui-rn?
I don't really like patching our own libraries since we do update them quite often (unlike 3rd party dependencies) which can break the patch or the very least throw an annoying warning about mismatching versions.

File: src/components/themed/QrCode.js [hunk 2]
```diff
@@ -81,8 +81,5 @@ const getStyles = cacheStyles((theme: Theme) => ({
     position: 'absolute',
     right: 0,
     top: 0
-  },
-  surface: {
-    backgroundColor: theme.qrBackgroundColor
```
Comment (thehobbit85) [inline src/components/themed/QrCode.js:86]: Why was this removed? Wouldn't it change the way it looks?

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

File: patches/edge-login-ui-rn+0.9.32.patch (hunk 1 above)
Comment (swansontec) [reply patches/edge-login-ui-rn+0.9.32.patch:20]: Breaking change to the native deps, which affects third-party integrations. We would need a major version bump.

Comment (thehobbit85) [reply patches/edge-login-ui-rn+0.9.32.patch:20]: Good enough reason

**Review by thehobbit85 — APPROVED** [review]

**Review by thehobbit85 — APPROVED** [review]
